### PR TITLE
Loggers linting

### PR DESCRIPTION
**Pull request by MStoykov**

### CODE REVIEW

File: logger/buffers/impl.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+	b := &Buffers{
+		buffer: new(bytes.Buffer),
+	}
+	return b, nil
+}
+
+// Buffers writes all the logs to
+// bytes.Buffer for later examination
+type Buffers struct {
+	buffer *bytes.Buffer
```
Comment (na--) [inline logger/buffers/impl.go]: For this logger to be useful as a mock logger, we need some way to get the logged messages externally. Either this has to be exported or we need a getter method. Also, why use a buffer? Why not just a slice of strings that we append to?

File: logger/mock.go [hunk 2]
```diff
@@ -1,15 +1,12 @@
 package logger
 
 import (
-	"github.com/ironsmile/nedomi/logger/nillogger"
+	"github.com/ironsmile/nedomi/logger/buffers"
 	"github.com/ironsmile/nedomi/types"
 )
 
 // NewMock returns a mock logger instance that can be used for testing.
 func NewMock() types.Logger {
```
Comment (na--) [inline logger/mock.go]: The return type should probably be `*buffers.Buffers`?

File: logger/buffers/impl.go (hunk 1 above)
Comment (MStoykov) [inline logger/buffers/impl.go]: Primarily because this is the 6th implementation of this mock. I'm gonna make it an exported []string

**Comment by na--** [comment]
I think you can safely re-enable the linter check in travis here.

**Comment by na--** [comment]
:+1:

**Pull request merged**